src/partials/carousel.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="banner">
```

Comment:
Can we merge these 2 divs
<div class="banner">
<div id="carousel" class="carousel carousel-container">
and use Ul LI for 
<div class="slides">
       <div  class='item slide' data-state="active">